src/transport/net_ib.cc

```diff
@@ -783,6 +817,19 @@ ncclResult_t ncclIbInit(ncclDebugLogger_t logFunction, ncclProfilerCallback_t pr
     INFO(NCCL_INIT|NCCL_NET, "NET/IB : Using%s %s; OOB %s:%s", line, ncclIbRelaxedOrderingEnabled ? "[RO]" : "",
           ncclIbIfName, ncclSocketToString(&ncclIbIfAddr, addrline));
 
+    ncclIbUseInline = ncclParamIbUseInline();
+    ncclIbGdrFlushDisable = ncclParamIbGdrFlushDisable();
+
+    rcclAinic = ((rcclParamAinic() == 1) ? true : false);
+    if (rcclAinic) {
+      rcclCtsInlineData = ((rcclParamCtsInlineData() == -1) ? true : false);
+      rcclCtsOffloadEnabled = ((rcclParamCtsOffloadEnabled() == -1) ? true : false);
+      // for AINIC to disable Inline data, set env NCCL_IB_USE_INLINE=-1
+      ncclIbUseInline = ((ncclParamIbUseInline() == -1) ? false : true);
+      // for AINIC GDR flush is disabled by default
+      ncclIbGdrFlushDisable = 1;
```

Comment:
It worth to have such kind of debug output to confirm that AINIC support is enabled, and identify set of optimizations has been used. Otherwise it is hard to check actual settings used during a run.

```suggestion
      ncclIbGdrFlushDisable = 1;
      INFO(NCCL_INIT|NCCL_NET, "NET/IB : AINIC optimizations enabled: CTS Inline Data: %s; CTS Offload %s;  GDR Flush: disabled",
                rcclCtsInlineData ? "Enabled": "Disabled",
                rcclCtsOffloadEnabled ? "Enabled": "Disabled");                
```

Reply:
sure, will add the same

Reply:
comment addressed in latest commit.